cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -2198,7 +2204,7 @@ def test_obsm_values_no_X_embedding__non_spatial_dataset(self, validator_with_ad
         ]
         assert validator.is_spatial is False
         assert validator.warnings == [
-            "WARNING: Dataframe 'var' only has 4 rows. Features SHOULD NOT be filtered from expression matrix.",
+            # "WARNING: Dataframe 'var' only has 4 rows. Features SHOULD NOT be filtered from expression matrix.",
```

Comment:
can we just remove these instead of commenting them out?

Reply:
yes!